src/Docker/DatabaseServices.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/*
+ * This file is part of the Symfony MakerBundle package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\MakerBundle\Docker;
+
+/**
+ * @author  Jesse Rushlow <jr@rushlow.dev>
+ *
+ * @internal
+ */
+final class DatabaseServices
+{
+    public static function getDatabaseSkeleton(string $name, string $version): array
+    {
+        switch ($name) {
+            case 'mariadb':
+                return [
+                    'image' => sprintf('mariadb:%s', $version),
+                    'environment' => [
+                        'MYSQL_ROOT_PASSWORD' => 'password',
+                    ],
+                    'ports' => ['3306:3306'],
+                ];
+            case 'mysql':
+                return [
+                    'image' => sprintf('mysql:%s', $version),
+                    'environment' => [
+                        'MYSQL_ROOT_PASSWORD' => 'password',
+                    ],
+                    'ports' => ['3306:3306'],
+                ];
+            case 'postgres':
+                return [
+                    'image' => sprintf('postgres:%s', $version),
```

Comment:
Are no env vars needed for postgres! I know that Fabien includes these in his book:

```yml
POSTGRES_USER: main
POSTGRES_PASSWORD: main
POSTGRES_DB: main
```

Reply:
good catch, I think postgres needs a password.. i'll double check the docs and adjust accordingly..

Reply:
done